src/app/wizard/services/app-launcher-project-progress.service.ts

```diff
@@ -17,8 +17,12 @@ export class AppLauncherProjectProgressService implements ProjectProgressService
   constructor(private helperService: HelperService) {
     this.END_POINT = this.helperService.getBackendUrl();
     this.END_POINT = this.END_POINT.split('/api')[0];
-
-    if (this.END_POINT && (this.END_POINT.startsWith("/") || this.END_POINT.startsWith(":"))) {
+    if (this.END_POINT.indexOf('https') !== -1) {
```

Comment:
It's not really important, but why use `indexOf() !== -1` here when that can be ambiguous instead of `startsWith()` which is what you really want? (Now a URL with "https" in it somewhere else then at the start might be very rare, but it _might_ happen. More so perhaps with "wss")

Reply:
That makes sense. It needs to be  updated in fabric8-ui too, as I took this code from there.